### PR TITLE
added-typerwriter-ani-files

### PR DESCRIPTION
**Pull request by Rehan-Khan-Engineer**

### CODE REVIEW

File: tasks-by-users/Rehan-Khan-Engineer/09-Typewriter-animation/style.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
+:root {
+  --bg-color: rgb(214, 122, 16);
+  --type-writer-speed: 6s;
+  --type-writer-steps: 24;
+}
+
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/09-Typewriter-animation/style.css:6]: Add css reset @Rehan-Khan-Engineer

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

**Review by shaikh-7abish — APPROVED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/09-Typewriter-animation/style.css (hunk 1 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/09-Typewriter-animation/style.css:6]: done @Abbas4tech

**Review by nehalskdev — APPROVED** [review]

**Review by nehalskdev — APPROVED** [review]

**Review by Rehmatulla-Khan — APPROVED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/09-Typewriter-animation/style.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```
Comment (mmusk) [inline tasks-by-users/Rehan-Khan-Engineer/09-Typewriter-animation/style.css:10]: Add these properties to universal selector @Rehan-Khan-Engineer

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

**Review by mujeeb100 — APPROVED** [review]

**Comment by Rehmatulla-Khan** [comment]
close this as well @Rehan-Khan-Engineer

File: tasks-by-users/Rehan-Khan-Engineer/09-Typewriter-animation/style.css (hunk 2 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/09-Typewriter-animation/style.css:10]: added CSS reset to universal selector as a good practice. @Abbas4tech @mmusk

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]

**Review by syedabdullahali — APPROVED** [review]

**Review by mmusk — APPROVED** [review]

**Review by Abbas4tech — APPROVED** [review]

**Review by nehalskdev — APPROVED** [review]

**Pull request merged**